File: ms-medicacao/src/main/java/com/h_salvacao/ms_medicacao/model/Encaminhamento.java
package com.h_salvacao.ms_medicacao.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Encaminhamento {

    private Long id;
    private String numToken;
    private Long fichaId;
    private String nomePaciente;
    private List<Regiao> regioesRaiox;
    @JsonProperty(value = "medicacaoIntravenosa")
    private List<MedicacaoIntravenosa> listaMedicacoes;

}

File: ms-token-app/src/app/entity/encaminhamento.ts
import { MedicacaoVeia } from "./medicacaoVeia";
import { Regiao } from "./regiao";

export class Encaminhamento{
    id: number;
    numToken: string;
    fichaId: number;
    nomePaciente: String;
    regioesRaiox: Regiao[];
    medicacaoIntravenosa: MedicacaoVeia[];
}